project/SparkBuild.scala

```diff
@@ -1433,7 +1492,10 @@ object CopyDependencies {
           if (jar.getName.contains("spark-connect") &&
             !SbtPomKeys.profiles.value.contains("noshade-connect")) {
             Files.copy(fid.toPath, destJar.toPath)
-          } else if (jar.getName.contains("spark-protobuf") &&
+          } else if (jar.getName.contains("connect-client") &&
+            !SbtPomKeys.profiles.value.contains("noshade-protobuf")) {
```

Comment:
Just double check if you really meant to use `spark-protobuf` or it is a copy-past typo?
The values to these three jars are a bit magical.

Reply:
Thanks, there was a typo. Fixed.